client/post-editor/editor-discussion/index.jsx

```diff
@@ -136,7 +136,37 @@ export class EditorDiscussion extends React.Component {
 						disabled={ ! this.props.post }
 						onChange={ this.onChange }
 					/>
-					<span>{ this.props.translate( 'Allow Pingbacks & Trackbacks' ) }</span>
+					<span>
+						{ this.props.translate( 'Allow Pingbacks & Trackbacks' ) }
+						<InfoPopover
+							position="top right"
+							className="editor-ping_status__info"
```

Comment:
This classname, and the pre-existing `editor-comment_status__info` seem to not be following our [coding standards](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#classes). While I realize the new field is only following the pattern of the above field, it's a great reason to fix it for the above case too.

The reason why `editor-comment_status__info` is not valid is because in classnames:
* We should use only one component prefix - in our case it should be `editor-discussion`, otherwise the linter will complain.
* Component prefix should be separated from the element by a double underscore (`__`).
* Both the component prefixes and elements can contain only dashes (`-`) but not underscores (`_`).

As you'll see in a comment below, I suggest that we change `.editor-comment_status__info` to `.editor-discussion__status-info`, and use it for both the info popovers.

Let me know if you have any questions though!

Reply:
That's a great explanation. Thanks, Marin!